git_machete/client.py

```diff
@@ -1517,6 +1517,8 @@ def __pick_remote(self, branch: str, is_called_from_traverse: bool) -> None:
                    "repository, or 'q' to quit creating pull request: "
 
         ans = input(msg).lower()
+        if not is_called_from_traverse and not ans.isnumeric():
+            raise MacheteException('Could not establish remote repository, pull request creation inrupted.')
```

Comment:
```suggestion
            raise MacheteException('Could not establish remote repository, pull request creation interrupted.')
```

Reply:
Also, this message will also be display in case of `q` when coming from `github create-pr`, right?

Reply:
Yes, it will be raised also with `q`.